parm/fetch/stage_atm_cold.yaml

```diff
@@ -0,0 +1,19 @@
+untar:
+    tarball : "{{ ATARDIR }}/{{ cycle_YMDH }}/atm_cold.tar"
+    on_hpss: True
+    contents:
+        - gfs_ctrl.nc
+        {% for ftype in ["gfs_data", "sfc_data"] %}
+        {% for ntile in range(1, ntiles + 1) %}
+        - {{ ftype }}.tile{{ ntile }}.nc
+        {% endfor %} # ntile
+        {% endfor %} # ftype
+    destination: "{{ DATA }}"
+atmosphere_cold:
+    copy:
+        - ["{{ DATA }}/gfs_ctrl.nc", "{{ COMOUT_ATMOS_INPUT }}"]
+        {% for ftype in ["gfs_data", "sfc_data"] %}
+        {% for ntile in range(1, ntiles + 1) %}
+        - ["{{ DATA }}/{{ ftype }}.tile{{ ntile }}.nc", "{{ COMOUT_ATMOS_INPUT }}"]
+        {% endfor %} # ntile
+        {% endfor %} # ftype
```

Comment:
We don't need to the copy portion.  This will be handled by the `stage_ic` job.

```suggestion
```

Reply:
Actually, this file should just be removed.  When `ci/cases/pr/ATM_cold.yaml` is moved here, it will replace this.

Reply:
Removing parm/fetch/stage_atm_cold.yaml